### PR TITLE
[JP] Apply #424 changes to Japanese translation

### PR DESCRIPTION
**Pull request by hinamiyagk**

### CODE REVIEW

File: jp/lessons/basics/basics.md [hunk 1]
```diff
@@ -6,28 +6,46 @@ order: 1
...
+## 入門
 
 ### Elixirのインストール
 
 各OS向けのインストール方法は Elixir-lang.org 上の[Installing Elixir](http://elixir-lang.org/install.html) で探すことができます。
 
+Elixirがインストールされたら簡単にバージョンを確認できます。
+
+    % elixir -v
+    Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
```
Comment (doomspork) [inline jp/lessons/basics/basics.md]: @nscyclone could you explain to @hinamiyagk what needs to be changed here to support your last PR?

**Comment by doomspork** [comment]
This is great, thank you @hinamiyagk!  @3100 could you take a peek at this if you get a chance?

File: jp/lessons/basics/basics.md (hunk 1 above)
Comment (nscyclone) [inline jp/lessons/basics/basics.md]: Sure, @doomspork .
@hinamiyagk, we've just implemented a feature to update the OTP and erts versions easily in further. In order to use it `Erlang/OTP 18 [erts-7.3]` should be replaced with `Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}]`. Examples and description can be found in #441 if needed.

Comment (hinamiyagk) [inline jp/lessons/basics/basics.md]: I see. I'll resolve conflict carefully.

**Comment by hinamiyagk** [comment]
@3100 Could you review this PR if you don't mind?

**Comment by 3100** [comment]
@hinamiyagk Sure, of course. As @doomspork and @nscyclone mentioned, I suppose we should support #441. Is it possible to ask you to update your code for it?

**Comment by hinamiyagk** [comment]
I thought I should resolve conflict by merging master after review, but should I commit additionally to support #441 ?

**Comment by 3100** [comment]
@hinamiyagk I see. It would be helpful if you could resolve conflicts. 👍

File: jp/lessons/basics/basics.md (hunk 1 above)
Comment (3100) [inline jp/lessons/basics/basics.md]: This line should be modified like below:
>    Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

**Comment by 3100** [comment]
(WIP: I will remove this comment when I finish my review. )

File: jp/lessons/basics/basics.md [hunk 2]
```diff
@@ -6,28 +6,43 @@ order: 1
...
+    Elixir {{ site.elixir.version }}
+
 ### 対話モード
 
 Elixirには`iex`という対話シェルが付属しており、入力したそばからElixirの式を評価することができるようになっています。
 
 対話モードを開始するには、`iex`を起動しましょう:
 
-	Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
```
Comment (3100) [inline jp/lessons/basics/basics.md]: This is a bit different from the original (English) version:
```
    Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

    Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
    iex(1)>
```

Comment (hinamiyagk) [inline jp/lessons/basics/basics.md]: oh... I'll fix it.

**Comment by 3100** [comment]
@hinamiyagk Thank you for your update. I reviewed it and added some comments. I also find that [this](https://github.com/doomspork/elixir-school/commit/b6ab35f135d5e0ea6f6ab914f0eb428fb19f1b71) should be fixed at some time, however, I am leaving it to you.

**Comment by 3100** [comment]
Thank you!  Would you also check this? https://github.com/doomspork/elixir-school/pull/438/files#r66704984

**Comment by hinamiyagk** [comment]
Fixed.

**Comment by 3100** [comment]
@hinamiyagk I appreciate all your help.

**Pull request merged**